codelabs/Utopia Wholesale Codelab/Holder/0-Obtaining a credential.md

```diff
@@ -79,6 +79,7 @@ val OPENID4VCI_CLIENT_PREFERENCES = OpenID4VCIClientPreferences(
 ```
 
 ProvisioningSupport is a subclass of OpenID4VCIBackend, which is defined in the Multipaz library. ProvisioningSupport class is the bridge between your wallet and credential issuers. It handles authentication, authorization, and secure communication.
+Here creates an OPENID4VCI_CLIENT_PREFERENCES object, which defines configuration parameters such as clientId, redirectUrl, locals, and signingAlgorithms. The OPENID4VCI_CLIENT_PREFERENCES is then used when calling launchOpenID4VCIProvisioning.
```

Comment:
pls add a new line -- currently this is not split into a new paragraph.

<img width="1027" height="418" alt="Image" src="https://github.com/user-attachments/assets/aac451bc-8a23-4306-8453-f25c0c6fcf74" />

also use backticks when referring to code like `OPENID4VCI_CLIENT_PREFERENCES`, `redirectUrl` etc.